### ISSUE TITLE
Term Request: Sheet-like posterior corneal stromal opacities

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11748]
**Preferred term label:**
Sheet-like posterior corneal stromal opacities

**Synonyms**
* Posterior amorphous corneal stromal opacities
* Continuous posterior stromal hyperreflective band
* Sheet-like pre-Descemet stromal opacities

**Definition (free text, please give PubMed ID)**
Diffuse, gray-white, sheet-like or lamellar opacification predominantly involving the posterior layers of the corneal stroma and appearing on anterior segment OCT as a continuous hyperreflective band directly anterior to Descemet membrane, frequently accompanied by corneal thinning and flattening (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Opacification of the corneal stroma (HP:0007759)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Posterior amorphous corneal dystrophy (PACD) (ORPHA:98971, OMIM:612868)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
